lib/active_merchant/billing/gateways/shopify.rb

```diff
@@ -0,0 +1,78 @@
+require 'shopify_api'
+
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class ShopifyGateway < Gateway
+      self.homepage_url = 'https://shopify.ca/'
+      self.display_name = 'Shopify'
+
+      def initialize(options = {})
+        requires!(options, :api_key)
+        requires!(options, :password)
+        requires!(options, :shop_name)
+
+        @api_key = options[:api_key]
+        @password = options[:password]
+        @shop_name = options[:shop_name]
+
+        @refunder_class = options[:refunder] || default_refunder
+        @transaction_repository = options[:transaction_repository] || default_transaction_repository
+
+        init_shopify_api!
+
+        super
+      end
+
+      def void(transaction_id, options = {})
+        transaction = find_transaction(transaction_id, options[:order_id])
+
+        return_response create_refund(transaction.amount, transaction)
+      end
+
+      def refund(money, transaction_id, options = {})
+        transaction = find_transaction(transaction_id, options[:order_id])
+
+        return_response create_refund(money, transaction)
```

Comment:
Is the term 'return_response' what we are actually returning? Not `billing_response` or `transaction_response`?